### PR TITLE
Update evalcast description

### PR DESCRIPTION
**Pull request by ryantibs**
Hi all, particularly @jacobbien @bnaras @dajmcdon, this is just a trivial update to the evalcast DESCRIPTION file. I added Dan as an author.

Note: I think we should remove the "cre" role from Jacob, because [as per my understanding from here](https://r-pkgs.org/description.html), this is used to mark the maintainer, which is not Jacob at this point. Who should that be? Dan?  Naras? Whatever is decided, please go ahead and just make the necessary changes on a commit on this branch.

Also, is there anyone else to add as an author (or a contributor)? If so you can sneak it in another commit here on this branch.

### CODE REVIEW

**Comment by ryantibs** [comment]
I just added Jed and Mike as authors, and a few URLs.  @dajmcdon I added the following URL for the vignette: https://cmu-delphi.github.io/covidcast/evalcastR/ (to mirror the covidcast one: https://cmu-delphi.github.io/covidcast/covidcastR/). 

I just checked now and I see that the evalcast vignette is living on the evalcast branch, not the main branch.  Before it gets merged into main, we can rename the folder docs/evalcastR (rather than docs/evalcast) if we want to keep that link.  I don't have strong feelings, but I'll note that aside from mirroring the covidcast link, the postfix "R" leaves it open that there might be a Python version fo evalcast at some point in the future.

**Review by dajmcdon — APPROVED** [review]

**Comment by dajmcdon** [comment]
@ryantibs  I went ahead and switched the maintainer to me.

As for the vignette + pkgdown, it's all on `evalcast` at the moment. I went ahead and renamed `docs/evalcast` -> `docs/evalcastR` for consistency. Maybe we can discuss merging the `evalcast` branch to `main` tomorrow?

**Comment by jacobbien** [comment]
Yes I agree!

File: R-packages/evalcast/DESCRIPTION
Comment (jacobbien) [inline R-packages/evalcast/DESCRIPTION:22]: indentation seems off though.

File: R-packages/evalcast/DESCRIPTION
Comment (ryantibs) [inline R-packages/evalcast/DESCRIPTION:9]:
```suggestion
           role = "aut"),
```

File: R-packages/evalcast/DESCRIPTION
Comment (ryantibs) [inline R-packages/evalcast/DESCRIPTION:10]:
```suggestion
```

File: R-packages/evalcast/DESCRIPTION
Comment (ryantibs) [inline R-packages/evalcast/DESCRIPTION:22]:
```suggestion
		person(given = "Daniel",
			     family = "McDonald",
			     role = c("aut", "cre"),
			     email = "daniel@stat.ubc.ca"),
```

**Review by ryantibs — COMMENTED** [review]
@dajmcdon I update this a bit, to fix the formatting, remove Jacob's email, and add yours.

The indentation still seems off but this might just be a function of the fact that I made these suggestions through the GitHub browser.  Please take a look.

**Comment by ryantibs** [comment]
@JedGrabman @sgsmob I just requested a review from you guys so you can choose/confirm how you want to be attributed.  (E.g., as "aut" or "ctb".)

**Review by sgsmob — APPROVED** [review]

File: R-packages/evalcast/DESCRIPTION
Comment (JedGrabman) [inline R-packages/evalcast/DESCRIPTION:38]: At the moment, this link leads to a 404, but I think @dajmcdon is updating the docs?

**Review by JedGrabman — APPROVED** [review]
Approval for adding me as author.

File: R-packages/evalcast/DESCRIPTION
Comment (dajmcdon) [reply R-packages/evalcast/DESCRIPTION:38]: I updated pkg docs in #248 on the `evalcast` branch. I haven't done a PR against main yet to incorporate, but we can...

**Pull request closed without merging**
